style.css

```diff
@@ -38,6 +43,18 @@ a {
   grid-area: main;
 }
 
+div.modal.modal {
```

Comment:
Here we are referring to the div that has a class of modal what is the second .modal for?

Reply:
To increase the specificity, in the CL i was using `div.modal` so i had to increase it even more hence ended up using same css selector twice.